requirements.txt

```diff
@@ -8,7 +8,8 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+requests>=2.9.1
 
 # Python3 compat
 future>=0.15.2
-configparser>=3.3.0; python_version < '3'
+#configparser>=3.3.0; python_version < '3'
```

Comment:
The problem with messing up with this in the .spec is that we need to manage different cases depending of the version of setuptools adopted by the distribution. One way to solve that cleanly would be to use a PB macro in requirements.txt that will be instantiated based on the distro and thus create the right file. Or a macro in the spec doing this. I'll try that.